src/jupyter_contrib_nbextensions/nbextensions/codefolding/codefolding_editor.yaml

```diff
@@ -0,0 +1,12 @@
+Type: Jupyter Notebook Extension
+Name: Codefolding in Editor
+Description: |
+  Enables the CodeMirror feature to allow codefolding in the Jupyter file
+  editor view.
+  Note that this also uses the codefolding hotkey from the codefolding
+  nbextension for the notebook view.
```

Comment:
Didn't think of that. We should only have one hotkey. I don't think anyone would want different hotkeys in notebook and edit mode.